### DISCUSSION TITLE
Welcome to dui-iglesias-software-samuel Discussions!

### DISCUSSION THREAD

**Discussion by CriistiianDM** [CriistiianDM/dui-iglesias-software-samuel#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CriistiianDM** [comment]
Chicos estaremos usando este chat de discusiones para cuando tengan bugs y demás cosas temas acerca del desarrollo dejarlo por aqui para que nos acostumbremos mas al entorno real del desarrollo

**Reply by JFOZ1010** [reply]
Vamos a Programar!
